### ISSUE TITLE
Greyed text does not show in Blue Theme (Extra Contrast)

### ISSUE THREAD

**Issue by cartermp** [Microsoft/visualfsharp#3789]
![](https://user-images.githubusercontent.com/6309070/31837492-d5df835e-b58d-11e7-9f8d-07a05c570225.PNG)

Both `System` and `System.Threading` are greyed out in the other themes.

**Comment by cartermp** [comment]
Ah, this seems to be related to the theme.  Light grey atop a white editor most assuredly doesn't meet accessibility requirements.

**Comment by vasily-kirichenko** [comment]
How C# editor shows unused symbols? Same problem?

**Comment by cartermp** [comment]
@vasily-kirichenko unused is a warning for C#, so it's the green squiggle across all themes.

**Comment by vasily-kirichenko** [comment]
@cartermp great, I'll make it the same in my PR.
